trigger.js

```diff
@@ -206,6 +215,44 @@ class TriggerHappy {
         Hooks.once('updateToken', () => this._executeTriggers(triggers));
         return true;
     }
+  
+    _onPreUpdateToken(scene, id, update) {
+        if (!scene.isView) return true;
+        if (update.x === undefined && update.y === undefined) return true;
+        const token = scene.data.tokens.find(t => t._id === update._id);
+        if (token.hidden) return true; // don't stop ivnisible tokens?
+
+        // Get all trigger tokens in scene
+        let targets = this._getTriggerTokens(canvas.tokens.placeables, this.triggers, 'capture');
+        if (!targets) return true;
+
+        let x = update.x || token.x;
+        let  y = update.y || token.y;
+        let tw = token.width * canvas.scene.data.grid / 2;
+        let th = token.height * canvas.scene.data.grid / 2;
+        let motion = new Ray({x: token.x + tw, y: token.y + th}, {x: x + tw, y: y + th});
+
+        // don't trigger on tokens that are already captured
+        targets = targets.filter(target => token.x !== target.x + target.w  / 2 - tw || token.y !== target.y + target.h / 2 - th)

```

Comment:
I think you can use `token.center` here to make it a little easier to read. 
I think you're checking the token's x/y (upper left corner) with the center of the trigger shifted by the half width/height of the token... probably doing `token.center.x !== target.center.x || token.center.y !== token.center.y` would be easier to read.